package.json

```diff
@@ -121,16 +121,18 @@
     "typescript": "^4.7.3"
   },
   "scripts": {
-    "test": "NODE_CONFIG_ENV=test MODE=test jest --forceExit",
-    "test:ci": "cp ./resources/blockchain-test-key.json ./tests/resources/key.json && POINT_KEYSTORE=./tests/resources MODE=test jest --forceExit --reporters=default --reporters=jest-junit",
+    "pretest": "mkdir -p ./tests/resources/datadir/keystore && cp ./resources/blockchain-test-key.json ./tests/resources/datadir/keystore/key.json && npm run build",
+    "test": "npm run pretest && npm run build:explorer:dev && POINT_DATADIR=./tests/resources/datadir POINT_KEYSTORE=./tests/resources/datadir/keystore NODE_CONFIG_ENV=test MODE=test jest --forceExit",
```

Comment:
The `npm run pretest` is not intended to be run manually, and may actually cause errors. It's a [npm feature](https://docs.npmjs.com/cli/v8/using-npm/scripts#pre--post-scripts) and will run automatically.